app/views/documents/index/_filters.html.erb

```diff
@@ -100,6 +100,26 @@
     %>
   </div>
 
+  <div class="govuk-form-group">
+    <%= render "govuk_publishing_components/components/label", {
+      text: t("documents.index.filter.political"),
+      html_for: "political-filter",
+      bold: true
+    } %>
+
+    <%= select_tag "political",
+      options_for_select([["Yes", "include_political"], ["No", "exclude_political"]], [params[:political]]),
```

Comment:
Might be simpler to have these refer to yes or no rather than having the include_political and exclude_political, not sure why they'd need to differ from the defaults.

Reply:
We were basing it off the HTML in the prototype, but agreed, `yes`/`no` would be fine as values 👍

Reply:
That makes sense, we should be cautious of that since the HTML isn't wrote by a developer.

By the way on that note in the design for the history mode option there is text of "Yes - current government" and "No - past government" which don't seem to make sense as these yes and no don't associate with current or past government. So worth considering that as Yes/No too.

Reply:
Changed. cc'ing @gclssvglx just to keep him in the loop.

Graham, we changed this because a document may not be in history mode for a number of reasons, not just whether it's connected to a past government.